src/hupper/reloader.py

```diff
@@ -309,3 +309,16 @@ def start_reloader(
         logger=logger,
     )
     return reloader.run()
+
+
+def watch(*args, **kwargs):
+
+    def wrapper(func):
+        worker_path = '{}.{}'.format(func.__module__, func.__name__)
+        start_reloader(worker_path, **kwargs)
```

Comment:
This should check `get_reloader` first and `if not None` then raise an exception. Only functions that are called once (such as main functions) should be decorated as the reloader is not re-entrant right now. Invoking two functions in the same process that both call `start_reloader` probably won't do what's expected.

Reply:
@mmerickel thanks for your review.
I've just pushed a new commit with a check on `is_active`, is that OK?